docs/LSPSupport.md

```diff
@@ -19,7 +19,6 @@ Current state of [Text Document Synchronization](https://microsoft.github.io/lan
  * ✅ [textDocument/didSave](https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#textDocument_didSave).
  * ❌ [textDocument/willSave](https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#textDocument_willSave).
  * ❌ [textDocument/willSaveWaitUntil](https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#textDocument_willSaveWaitUntil).
- * ❌ [textDocument/didRename](https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#textDocument_didRename).

```

Comment:
If didRename is now supported it should be listed as such. If not, this line should be restored as such

Reply:
the `textDocument/didRename` doesn't exist (it is a strange link). It is `textDocument/rename` and `textDocument/prepareRename` which exists that we use and it is listed in the LSP support page.